src/services/userService.ts

```diff
@@ -48,3 +48,7 @@ export const updateUser = async (args: IUser): Promise<IUser> => {
     })
     .catch((error: Error) => Promise.reject(Error(`Unable to update user. Error: ${error}`)));
 };
+
+export const deleteUser = async (userId: string): Promise<void> => {
+  removeUser(userId).catch((error: Error) => Promise.reject(Error(`Unable to delete user with id: ${userId}. Error: ${error}`)));
```

Comment:
translate missing

Reply:
we are not showing this to a user, and we are not translating any other error. do you want me to add this?